swagger.yaml

```diff
@@ -1167,3 +1238,15 @@ components:
         value:
           type: boolean
           description: Boolean value to indicate if option is correct or incorrect
+
+    FlextimeModel:
+      type: object
+      properties:
```

Comment:
Consider whether we should also return the userSeveraId as part of this schema- it's probably not necessary but still worth considering. Rememeber we are not bound to have this be identical to what the serveraAPI returns.

Reply:
Would this be used when Admin wants to list employees flextime. Like there would return severaId and balances ?